app/experimenter/static/js/components/DesignForm.js

```diff
@@ -28,16 +29,21 @@ class DesignForm extends React.PureComponent {
     };
   }
 
+  isMultiPref() {
+    return this.state.data.get("is_multi_pref", this.props.isMultiPref);
+  }
+
   isBranchedAddon() {
-    return (
-      this.state.data.get("is_branched_addon") || this.props.isBranchedAddon
-    );
+    return this.state.data.get("is_branched_addon", this.props.isBranchedAddon);
```

Comment:
Also these both suffer from the bug I'm fixing in my pr, but don't worry I'll fix it.